src/client/components/BaselayerControl.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <button
+    class="baselayer-control"
+    @click="switchBaseLayer"
+  >
+    <span class="a11y-sr-only">Switch to {{ nextLayer.title }}</span>
+    <img
+      :src="nextLayer.thumbnail"
+      :alt="nextLayer.title"
+      class="baselayer-control__image"
+    >
+  </button>
+</template>
+
+<script>
+export default {
+  props: {
+    layers: {
+      type: Array,
+      required: true,
+      validator (layers) {
+        return layers.reduce((isValid, layer) => {
```

Comment:
I’d use `.every` here, it is more descriptive imho

Reply:
good one